index.html

```diff
@@ -0,0 +1,160 @@
+            </li>
+            <li>
+              <a href="#">News</a>
+            </li>
+            <li>
+              <a href="#">WM campaign</a>
+            </li>
+          </ul>
+          <div class="remaining"></div>
+        </nav>
+      </header>
+      <section id="home">
+        <p class="title-3">"Hello! Website developers"</p>
+        <h1 class="title-1">
+          <img src="Images/pictures/Main-text.svg" alt="" />
+        </h1>
+        <div class="card">
+          <small>
+            Our tutorials and content help students learn the ins and outs of
+            Web development. These tutorials are tailored to students of
+            different levels.
+          </small>
+        </div>
+        <h1 class="title-1">2022.3.7(Mon) ~ (Today)</h1>
+        <p>&copy;WinMac website development school. by Tadesse Alemayehu</p>
+      </section>
+    </main>
+    <section id="main-tutors">
+      <h2 class="title-1">Main Classe's</h2>
+      <hr class="short" />
+      <div class="card-list">
+        <div class="card">
+          <span class="icon">
+            <img src="./Images/Icons/html_5.svg" alt="" />
+          </span>
+          <h2 class="title-2">HTML</h2>
+          <p class="title-3">
+            Build a strong foundation in web development by learning JavaScript,
+            one of the major programming languages on the web.
+          </p>
+        </div>
+        <div class="card">
+          <span class="icon">
+            <img src="./Images/Icons/css3.svg" alt="" />
+          </span>
+          <h2 class="title-2">CSS</h2>
+          <p class="title-3">
+            Build a strong foundation in web development by learning JavaScript,
+            one of the major programming languages on the web.
+          </p>
+        </div>
+        <div class="card">
+          <span class="icon">
+            <img src="./Images/Icons/javascript.svg" alt="" />
+          </span>
+          <h2 class="title-2">Java Script</h2>
+          <p class="title-3">
+            Build a strong foundation in web development by learning JavaScript,
+            one of the major programming languages on the web.
+          </p>
+        </div>
+        <div class="card">
+          <span class="icon">
+            <img src="./Images/Icons/react_native.svg" alt="" />
+          </span>
+          <h2 class="title-2">React</h2>
+          <p class="title-3">
+            Build a strong foundation in web development by learning JavaScript,
+            one of the major programming languages on the web.
+          </p>
+        </div>
+        <div class="card">
+          <span class="icon">
+            <img src="./Images/Icons/nodejs.svg" alt="" />
+          </span>
+          <h2 class="title-2">Node.js</h2>
+          <p class="title-3">
+            Build a strong foundation in web development by learning JavaScript,
+            one of the major programming languages on the web.
+          </p>
+        </div>
```

Comment:
Due to semantics reasons, each class card should be wrapped in a `article` tag 🧐 Kindly change your divs into articles 😁

Reply:
@mateo951 Thank you. ill amend the changes now.